src/release/breaking-changes/material-state.md

```diff
@@ -0,0 +1,106 @@
+---
+title: Rename MaterialState to WidgetState
+description: >-
+  MaterialState and it's related APIs have been moved
+  outside of the Material library and renamed to
+  WidgetState.
+---
+
+## Summary
+
+`MaterialState`, and its related APIs have been moved out
+of the Material library and renamed to `WidgetState`.
+
+## Background
+
+`MaterialState` provided logic for handeling multiple different
+states a widget could have, like "hovered", "focused", and
+"disabled". Because this functionality could be useful outside
+of the Material library, namely the base Widgets layer and
+Cupertino, it was decided to move it outside of Material. As
+part of the move, and to avoid future confusion, the different
+MaterialState classes have been renamed to `WidgetState`. The
+behavior of the two are the same.
+
+| Before    | Now |
+| -------- | ------- |
+| MaterialState | WidgetState |
+| MaterialStatePropertyResolver | WidgetStatePropertyResolver |
+| MaterialStateColor | WidgetStateColor |
+| MaterialStateMouseCursor | WidgetStateColorMouseCursor |
+| MaterialStateBorderSide | WidgetStateBorderSide |
+| MaterialStateOutlinedBorder | WidgetStateOutlinedBorder |
+| MaterialStateTextStyle | WidgetStateTextStyle |
+| MaterialStateProperty | WidgetStateProperty |
+| MaterialStatePropertyAll | WidgetStatePropertyAll |
+| MaterialStatesController | WidgetStatesController |
+| MaterialStateMixin | WidgetStateMixin |
+
+The classes `MaterialStateOutlineInputBorder` and
+`MaterialStateUnderlineInputBorder` were left in the
+Material library with no `WidgetState` equivilant, as
+they are Material design specific.
+
+## Migration guide
+
+A [Flutter fix][] is available to help migrate the `MaterialState`
+classes to `WidgetState`.
+
+To migrate, replace `MaterialState` with `WidgetState`.
+
+Code before migration:
+
+```dart
+MaterialState selected = MaterialState.selected;
+
+final MaterialStateProperty<Color> backgroundColor;
+
+class _MouseCursor extends MaterialStateMouseCursor{
+    const _MouseCursor(this.resolveCallback);
+
+    final MaterialPropertyResolver<MouseCursor?> resolveCallback;
+
+    @override
+    MouseCursor resolve(Set<MaterialState> states) => resolveCallback(states) ?? MouseCursor.uncontrolled;
+}
+
+MaterialStateBorderSide? get side;
+```
+
+Code after migration:
+
+```dart
+WidgetState selected = WidgetState.selected;
+
+final WidgetStateProperty<Color> backgroundColor;
+
+class _MouseCursor extends WidgetStateMouseCursor{
+    const _MouseCursor(this.resolveCallback);
+
+    final WidgetPropertyResolver<MouseCursor?> resolveCallback;
+
+    @override
+    MouseCursor resolve(Set<WidgetState> states) => resolveCallback(states) ?? MouseCursor.uncontrolled;
+}
+
+WidgetStateBorderSide? get side;
+```
+
+## Timeline
+
+Landed in version: xxx<br>
```

Comment:
@MitchellGoodwin, do you have the info on which main or beta release has this change? Put that here.

Reply:
It's not landed yet, we are trying to make sure our communication is ready to go before landing it. Should I put what release it's likely to be in for now? Or leave it as a placeholder.

Reply:
Leave it for now, but once you know, we should update the page.  thx!